lib/postgrex/protocol.ex

```diff
@@ -176,6 +178,26 @@ defmodule Postgrex.Protocol do
     end
   end
 
+  @spec handle_execute(Postgrex.Stream.t, list, Keyword.t, state) ::
+    {:ok, Postgrex.Result.t, state} |
+    {:error, ArgumentError.t, state} |
+    {:error | :disconnect, Postgrex.Error.t, state}
+  def handle_execute( %Postgrex.Stream{ query: query } = stream, params, options, state ) do
+    %{buffer: buffer} = state
+    status = %{ notify: notify( options ), sync: :sync, prepare: :parse_execute }
+    case query_member?( state, query ) do
```

Comment:
need to `execute_send` the stream ( https://github.com/ericmj/postgrex/pull/176/files#diff-3e2400d928ee8dfffbe041e24515a8b2R680 ) and `execute_recv` the stream 

not sure now how to reduce duplication ...